dev/small-requirements.txt

```diff
@@ -8,3 +8,20 @@ pytest-localserver==0.5.0
 moto==1.3.16
 # Test plugin, used to verify correctness of MLflow plugin APIs
 tests/resources/mlflow-test-plugin/
+
+# Required by mlflow.keras
+keras==2.3.1
+# Required by mlflow.sklearn
+scikit-learn==0.23.2
+# Required by mlflow.gluon
+mxnet==1.5.0
+# Required by mlflow.fastai
+fastai==1.0.60
+# Required by mlflow.tensorflow
+tensorflow==1.15.2
+# Required by mlflow.xgboost
+xgboost>=0.82
+# Required by mlflow.lightgbm
+lightgbm==2.3.0
+# Required by mlflow.spark
+pyspark==3.0.0
```

Comment:
Our small tests shouldn't depend on heavyweight ML libraries. Can we instead decorate autologging test cases as `@pytest.mark.large` so they're executed as part of our large suites?

Reply:
i think the tests are failing because import tensorflow is at the top of the file, so when running the small tests in the file tensorflow is not installed.

it seems to fix this we would either have to extract out the autologging test cases, or mark all the tests in the file as large (which I dont think we want)